yarn-project/aztec-nr/aztec/src/history/public_value_inclusion.nr

```diff
@@ -0,0 +1,44 @@
+use dep::protocol_types::constants::{
+    PUBLIC_DATA_TREE_HEIGHT,
+    GENERATOR_INDEX__PUBLIC_LEAF_INDEX,
+};
+use dep::std::merkle::compute_merkle_root;
+
+use crate::{
+    context::PrivateContext,
+    hash::pedersen_hash,
+    oracle::get_sibling_path::get_sibling_path,
+};
+
+pub fn prove_public_value_inclusion(
```

Comment:
This one should probably have an issue for fixing it when the underlying tree structure is changed from sparse trees in a few days.

Reply:
This will break when the tree is changed and it will be caught by the test so I feel like it's part of the [tree structure change issue](https://github.com/AztecProtocol/aztec-packages/issues/501). Won't you agree?